shibboleth-cluster-windows/azuredeploy.json

```diff
@@ -131,442 +63,416 @@
     }
   },
   "variables": {
-    "newStorageAccountName": "[concat(uniquestring(resourceGroup().id), 'shc')]",
-    "imagePublisher": "MicrosoftWindowsServer",
-    "imageOffer": "WindowsServer",
-    "OSVersion": "[parameters('windowsOSVersion')]",
-    "OSDiskName": "[concat(parameters('uniqueNamePrefix'),'Disk')]",
-    "nicName": "[concat(parameters('uniqueNamePrefix'),'Nic')]",
-    "addressPrefix": "10.0.0.0/16",
+    "vnetAddressRange": "10.0.0.0/16",
+    "subnetAddressRange": "10.0.0.0/24",
     "subnetName": "Subnet",
-    "subnetPrefix": "10.0.0.0/24",
-    "subnetNameDB": "SubnetDB",
-    "subnetPrefixDB": "10.0.1.0/24",
+    "availabilitySetName": "AvSet",
+    "vmName": "VM",
     "storageAccountType": "Standard_LRS",
-    "publicIPAddressName": "[concat(parameters('uniqueNamePrefix'),'IP')]",
-    "publicDBIPAddressName": "[concat(parameters('uniqueNamePrefix'),'DBIP')]",
-    "publicIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPAddressName'))]",
-    "publicIPAddressType": "Dynamic",
-    "vmStorageAccountContainerName": "vhds",
-    "vmName": "[concat(parameters('uniqueNamePrefix'),'VM')]",
-    "virtualNetworkName": "[concat(parameters('uniqueNamePrefix'),'VNet')]",
-    "subnetRef": "[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('virtualNetworkName'), variables('subnetName'))]",
-    "subnetRefDB": "[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('virtualNetworkName'), variables('subnetNameDB'))]",
-    "availabilitySetName": "[concat(parameters('uniqueNamePrefix'),'AvSet')]",
-    "lbName": "[concat(parameters('uniqueNamePrefix'),'LB')]",
-    "lbID": "[resourceId('Microsoft.Network/loadBalancers',variables('lbName'))]",
-    "frontEndIPConfigID": "[concat(variables('lbID'),'/frontendIPConfigurations/loadBalancerFrontend')]",
-    "lbProbeID": "[concat(variables('lbID'),'/probes/tcpProbe')]",
-    "lbPoolID": "[concat(variables('lbID'),'/backendAddressPools/LoadBalancerBackend')]",
+    "nicsql": "[concat(variables('vmName'),'sql')]",
+    "newStorageAccountName": "[concat('st', uniqueString(resourceGroup().id))]",
+    "subnet-id": "[concat(resourceId('Microsoft.Network/virtualNetworks','VNET'),'/subnets/',variables('subnetName'))]",
+    "imagePublisher": "MicrosoftWindowsServer",
+    "imageOffer": "WindowsServer",
+    "imageSku": "2012-R2-Datacenter",
     "installScriptName": "install_shibboleth_idp.ps1",
     "installScriptUri": "[concat('https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/shibboleth-cluster-windows/', variables('installScriptName'))]",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts